src/Catalyst.Node.POA.CE/Program.cs

```diff
@@ -0,0 +1,201 @@
+            new Dictionary<Type, Func<IModule>>
+            {
+                {typeof(CoreLibProvider), () => new CoreLibProvider()},
+                {typeof(MempoolModule), () => new MempoolModule()},
+                {typeof(ConsensusModule), () => new ConsensusModule()},
+                {typeof(KvmModule), () => new KvmModule()},
+                {typeof(LedgerModule), () => new LedgerModule()},
+                {typeof(HashingModule), () => new HashingModule()},
+                {typeof(DiscoveryHastingModule), () => new DiscoveryHastingModule()},
+                {typeof(RpcServerModule), () => new RpcServerModule()},
+                {typeof(BulletProofsModule), () => new BulletProofsModule()},
+                {typeof(KeystoreModule), () => new KeystoreModule()},
+                {typeof(KeySignerModule), () => new KeySignerModule()},
+                {typeof(DfsModule), () => new DfsModule()},
+                {typeof(AuthenticationModule), () => new AuthenticationModule()},
+                {
+                    typeof(ApiModule),
+                    () => new ApiModule("http://*:5005", new List<string> {"Catalyst.Core.Modules.Web3"})
+                },
+                {typeof(PoaConsensusModule), () => new PoaConsensusModule()},
+                {typeof(PoaP2PModule), () => new PoaP2PModule()},
+            };
+
+        public static void RegisterNodeDependencies(ContainerBuilder containerBuilder,
+            List<IModule> extraModuleInstances = default,
+            List<Type> excludedModules = default)
+        {
+            // core modules
+            containerBuilder.RegisterType<CatalystNodePoa>().As<ICatalystNode>();
+            containerBuilder.RegisterType<ConsoleUserOutput>().As<IUserOutput>();
+            containerBuilder.RegisterType<ConsoleUserInput>().As<IUserInput>();
+
+            // message handlers
+            containerBuilder.RegisterAssemblyTypes(typeof(CoreLibProvider).Assembly)
+               .AssignableTo<IP2PMessageObserver>().As<IP2PMessageObserver>();
+
+            containerBuilder.RegisterAssemblyTypes(typeof(RpcServerModule).Assembly)
+               .AssignableTo<IRpcRequestObserver>().As<IRpcRequestObserver>()
+               .PublicOnly();
+
+            // DAO MapperInitialisers
+            containerBuilder.RegisterAssemblyTypes(typeof(CoreLibProvider).Assembly)
+               .AssignableTo<IMapperInitializer>().As<IMapperInitializer>();
+            containerBuilder.RegisterType<MapperProvider>().As<IMapperProvider>()
+               .SingleInstance();
+
+            var modulesToRegister = DefaultModulesByTypes
+               .Where(p => excludedModules == null || !excludedModules.Contains(p.Key))
+               .Select(p => p.Value())
+               .Concat(extraModuleInstances ?? new List<IModule>());
+
+            foreach (var module in modulesToRegister) containerBuilder.RegisterModule(module);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)